modelexpress_common/src/cache.rs

```diff
@@ -208,6 +208,26 @@ impl CacheConfig {
         }
     }
 
+    /// Convert a Hugging Face folder name back to the original model ID
+    /// Examples:
+    /// - "models--google-t5--t5-small" -> "google-t5/t5-small"
+    pub fn folder_name_to_model_id(folder_name: &str) -> String {
+        // Handle models
+        if let Some(stripped) = folder_name.strip_prefix("models--") {
+            // Convert models--owner--repo to owner/repo
+            stripped.replace("--", "/")
+        } else if folder_name.starts_with("datasets--") {
+            // TODO: Handle datasets names conversion
+            folder_name.to_string()
+        } else if folder_name.starts_with("spaces--") {
+            // TODO: Handle spaces names conversion
+            folder_name.to_string()
+        } else {
+            // If it doesn't match the expected pattern, return as-is
+            folder_name.to_string()
+        }
```

Comment:
maybe just group all the other cases into `else` since they're all returned as-is for now?
```suggestion
        } else {
            // TODO: Handle other prefixes. return as-is for now.
            folder_name.to_string()
        }
```

Reply:
These are huggingface folder categories, just left as place holders😆

>is it common to use "folder" as varname in a program? i feel like folder is usually used in the context of gui but i guess it doesn't matter much 😆

the original function prints out a list of folder names, and the nvbug asks for hf id, so here are these changes🙃